### DISCUSSION TITLE
about train imagenet 1k

### DISCUSSION THREAD

**Discussion by frischzenger** [apache/incubator-mxnet#15238] [Q&A]
i have convert the imagenet images to the *.rec files by im2rec in the official tutorial, and the train with resnet 50 by default arguments. and after 80 epochs the train accuracy is 99.999% near to 1, but the validation accuracy is 0.11%, what's the reason cause this result? thanks

**Comment by zachgk** [comment]
It sounds like your model is suffering from overfitting (https://www.d2l.ai/chapter_multilayer-perceptrons/underfit-overfit.html)

**Comment by frischzenger** [comment]
@zachgk
the model is not a self-implementation,I just use the official released model.
and i changed to another official released model inception-v3 also get the same wrong result. i think maybe some thing wrong with the training procedure.

**Comment by piyushghai** [comment]
@frischzenger Can you share your training script ? It will help to debug it better.

**Comment by leleamol** [comment]
@frischzenger can you please share the commandline that you are using to train the model?

I you are using "train_imagenet.py" script in the github repository and setting "--benchmark 1", then the script will use the synthetic data (randomly generated training data) instead of imagenet dataset.

@mxnet-label-bot add [Pending Requester Info]

**Comment by frischzenger** [comment]
@leleamol 
python train_imagenet.py --data-train ./train/imagenet_train.rec --data-val ./val/imagenet_val.rec --network resnet --num-layers 50 --batch-size 256 --gpus '0,1,2,3'

**Comment by frischzenger** [comment]
i have found the error was caused by wrong image label. so I correct the right labels and now train is  78,but the val is 69% still lower than the offical val 76%. what is the reason cause this?

**Comment by leleamol** [comment]
@frischzenger data scientists and experts would be able to guide you more precisely. I would recommend asking the question on fourm here https://discuss.mxnet.io/   and seek guidance.

Couple of techniques that can help include adjusting resolution to 480px, performing training in multiple (2) steps with varying learning rates, etc.
I would recommend closing this issue after opening the discussion on MXNet forum.

**Comment by leleamol** [comment]
@mxnet-label-bot add [Training, Question]
